### ISSUE TITLE
Identify and skip already downloaded packages before downloading

### ISSUE THREAD

**Issue by yehancha** [RossMetacraft/WorldOfAiDownloader#4]

**Comment by RossCarlson1971** [comment]
I decided not to try to skip already-downloaded packages since it would require doing a checksum on the file to ensure it wasn't changed or corrupted since the previous download. I didn't feel it was worth the effort.

**Comment by yehancha** [comment]
Cannot we skip packages by their names? I'm really new to this WOA packages so please don't mind if I'm being inconvenient.

**Comment by RossCarlson1971** [comment]
You could, but as I said, if the previous download was corrupted, or incomplete, or the package has changed since the previous download, you still want to download a new one.

**Comment by yehancha** [comment]
Aha, got it now. I was thinking just to give the user an option to skip previously downloaded files by name. The user should use the option with his own risk. We can show him a message about the risk when taking the option.

**Comment by RossCarlson1971** [comment]
Yeah, that sounds like a good idea. If you're a coder, feel free to submit a pull request for the change, otherwise I'll do it when I find the time, or someone else could do it.

**Comment by yehancha** [comment]
Yes, I'm a developer. I will surely look into it as soon as I got time. Nice discussion, Ross, thanks!
